.gitignore

```diff
@@ -0,0 +1,7 @@
+
+# Build results
+bin/
+obj/
+/src/Temporalio/Bridge/target
+/src/Temporalio/Bridge/Cargo.lock
```

Comment:
I wouldn't gitignore `Cargo.lock` in this project.

Reply:
Agreed, this is a mistake by me. Will fix.

Reply:
+1

Reply:
Fixed